### ISSUE TITLE
Can't install because missing dependencies.

### ISSUE THREAD

**Issue by Zekodon** [ch1x0r/MintyFresh2016#1]
I'm pretty sure this isn't your software's fault but I got this error: 
![screenshot_2016-02-03_17-45-34](https://cloud.githubusercontent.com/assets/9424303/12799674/15e4f65a-ca9e-11e5-9baf-e80aeaebebc3.png)

I've been searching around synaptic but the package for live-config is broken or something. How do I install the dependency?

**Comment by ch1x0r** [comment]
So, here's the question - what version of sysvinit (if any) do you have? needs to be >=2.86
If you don't have sysvinit and are using upstart... there is no live-config.

I want to check on the system it was working on (older, might not use upstart) what is going on there.

**Comment by Zekodon** [comment]
I placed 'sysvinit' in synaptic's search field and I got these results. 
![screenshot_2016-02-10_21-16-18](https://cloud.githubusercontent.com/assets/9424303/12968149/ed2e634c-d03b-11e5-8f0b-c4cdfdfdf503.png)

However, whenever I try to install any live-config package (or derivatives) It doesn't allow me. Also I'm sure that I don't actually have sysvinit installed, so how would I install sysvinit? 

(you're able to see the images right?)
